packages/redux-store/package.json

```diff
@@ -29,12 +29,12 @@
     "react-redux": "^7.2.5"
   },
   "devDependencies": {
-    "@aries-framework/core": "^0.2.0",
+    "@aries-framework/core": "^0.4.0-alpha.87",
```

Comment:
Would be good to go to the latest for now to make the move to 0.4.0 stable easier.

https://www.npmjs.com/package/@aries-framework/core/v/0.4.0-alpha.104

Reply:
sure! nice catch. Didn't spot that because I just looked at the unresolved commenst from Jim's PR - my bad